### PR TITLE
[NEXUS-8343] expose OSS license details

### PR DESCRIPTION
**Pull request by jdillon**
https://issues.sonatype.org/browse/NEXUS-8343

related:

https://github.com/sonatype/nexus-bundles/pull/148

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Sonatype Nexus (TM) Open Source Version.
+Copyright (c) 2008-2015 Sonatype, Inc. All rights reserved.
```
Comment (mosabua) [inline NOTICE.txt]: Might want to change that to 2088-present like in other changes recently.

**Comment by jdillon** [comment]
@mosabua There is a separate story for that change

**Comment by adreghiciu** [comment]
+1

**Comment by cstamas** [comment]
+1

**Pull request merged**